js/charts/clusteredverticalbar.js

```diff
@@ -612,77 +646,104 @@ function clusteredverticalbar() {
 
     var drawViz = function (element) {
         var me = this;
-
-        element.append('rect')
-            .attr("x", function (d) {
-                return 0;
-            })
-            .attr("y", function (d) {
-                return y(d[d.measure]);
-            })
-            .attr('class', 'bar')
-            .attr("width", x1.bandwidth())
-            .attr("height", function (d) {
-                return 0;
-            })
-            .style('fill', function (d, i) {
-                return UTIL.getDisplayColor(_measure.indexOf(d.measure), _displayColor);
-            })
-            .style('stroke', function (d, i) {
-                return UTIL.getBorderColor(_measure.indexOf(d.measure), _borderColor);
-            })
-            .style('stroke-width', 2)
-            .on('mouseover', _handleMouseOverFn.call(chart, tooltip, _local_svg))
-            .on('mousemove', _handleMouseMoveFn.call(chart, tooltip, _local_svg))
-            .on('mouseout', _handleMouseOutFn.call(chart, tooltip, _local_svg))
-            .on('click', function (d) {
-                if ($("#myonoffswitch").prop('checked') == false) {
-                    $('#Modal_' + $(div).attr('id') + ' .measure').val(d.measure);
-                    $('#Modal_' + $(div).attr('id') + ' .threshold').val('');
-                    $('#Modal_' + $(div).attr('id') + ' .measure').attr('disabled', true);;
-                    $('#Modal_' + $(div).attr('id')).modal('toggle');
-                }
-                else {
-                    var confirm = d3.select(div).select('.confirm')
-                        .style('visibility', 'visible');
-                    var _filter = _Local_data.filter(function (d1) {
-                        return d[_dimension[0]] === d1[_dimension[0]]
-                    })
-                    var rect = d3.select(this);
-                    if (rect.classed('selected')) {
-                        rect.classed('selected', false);
-                        filterData.map(function (val, i) {
-                            if (val[_dimension[0]] == d[_dimension[0]]) {
-                                filterData.splice(i, 1)
-                            }
+        if (!_print) {
```

Comment:
the element rendering code can be kept outside since it is common for both frontend and server side rendering. Only the code for the mouse events can be kept inside the if block for "_print" as mouse events doesnot work in server side rendering